src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -6336,6 +6336,12 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						} else {
 							sofia_clear_pflag(profile, PFLAG_ALWAYS_BRIDGE_EARLY_MEDIA);
 						}
+					} else if (!strcasecmp(var, "telnyx-sip-proxy-timeout-hangup-cause")) {
+						if (switch_true(val)) {
+							sofia_set_pflag(profile, PFLAG_SIP_PROXY_TIMEOUT_HANGUP_CAUSE);
```

Comment:
I think its better to be a string that is translated to switch cause. This will be helpful to change later on the error cause if decided to change it.